config/configuration.go

```diff
@@ -15,6 +15,7 @@ const (
 	SocketPrivateKeyFile     string = "SocketPrivateKeyFile"
 	SocketCertificateFile    string = "SocketCertificateFile"
 	SocketCAFile             string = "SocketCAFile"
+	SocketTLSForTesting      string = "SocketTLSForTesting"
```

Comment:
This configuration name wasn't clear to me until I dug into the code.  The TLS config "InsecureSkipVerify" is well named, perhaps we could use that somewhere in the configuration name? Something like `SocketTLSInsecureSkipVerify`?